doc/classes/Transform2D.xml

```diff
@@ -270,15 +271,16 @@
 			# | 1 | 0 | 0
 			# | 0 | 1 | 0
 			[/codeblock]
-			This is identical to creating [constructor Transform2D] without any parameters. This constant can be used to make your code clearer, and for consistency with C#.
+			If a [Vector2], a [Rect2], or another [Transform2D] is transformed (multiplied) by this constant, no transformation occurs.
```

Comment:
Also `PackedVector2Array`.

Reply:
A bit nitpicky since for packed arrays it ends up operating on the individual vectors, but since we are mentioning _all_ Variant types affected, it makes sense to be precise, I suppose.

Reply:
True, but potential additional benefit is making users more aware that they can apply transforms to packed arrays. I've seen many times people manually using loops in GDScript instead of using a single `Transform * PackedVectorArray` multiplication (which results in shorther code but is also faster).